everyvoice/cli.py

```diff
@@ -303,6 +303,18 @@ def demo(
         file_okay=True,
         autocompletion=complete_path,
     ),
+    languages: str = typer.Option(
+        "all",
+        "--languages",
+        "-l",
+        help="Without spaces and separated by commas, specify the list of languages which will be available in the demo. Example list: eng,fin",
```

Comment:
I think the preferred way to do this in typer is like this:

```
languages: list[str] = typer.Option(
        ['all'],
        "--languages",
        "-l",
        help="Specify languages to be included in the demo. Example: everyvoice demo <path_to_text_to_spec_model> <path_to_spec_to_wav_model> --languages eng --languages fin",
    ),
```

The `list[str]` typing means that typer will automatically return a list, and handle things like extra whitespace, see https://typer.tiangolo.com/tutorial/multiple-values/multiple-options/